src/CalculatorInterface.php

```diff
@@ -0,0 +1,78 @@
+<?php
+namespace Money;
+
+/**
+ * Interface CalculatorInterface
+ * @package Money
+ */
+interface CalculatorInterface
+{
+
+    /**
+     * @return bool
+     */
+    public function supported();
```

Comment:
Can't we make this a static function? In that case we don't need to instantiate all calculators when detecting the calculator.